typing/jkind.ml

```diff
@@ -1479,7 +1488,7 @@ let for_boxed_record lbls =
     in
     add_labels_as_baggage lbls base
 
-(* CR layouts v2.8: This should take modalities into account. *)
+(* XCR layouts v2.8: This should take modalities into account. *)
```

Comment:
Not sure what you meant by `XCR` here. This still (as I'm sure you know) does not take modalities into account. I've added this ability, though, in a commit I'm about to push.

Reply:
oop, yeah, I meant to use this as a reminder to myself to update this, but forgot to before pushing. thanks!